package.json

```diff
@@ -26,19 +28,20 @@
     "eslint-plugin-react": "^7.30.0",
     "eslint-plugin-react-hooks": "^4.3.0",
     "husky": "^7.0.4",
-    "i18next": "^21.5.2",
+    "i18next": "^21.10.0",
     "jest": "^28.1.1",
     "jest-environment-jsdom": "^28.1.1",
     "lerna": "^4.0.0",
     "lint-staged": "^12.0.2",
     "postcss": "^8.3.8",
     "prettier": "^2.4.1",
-    "react": "^17.0.2",
-    "react-dom": "^17.0.2",
-    "react-i18next": "^11.14.2",
-    "storybook-addon-apollo-client": "^4.0.9",
+    "react": "^18.2.0",
+    "react-dom": "^18.2.0",
+    "react-i18next": "^11.18.6",
+    "storybook": "^6.5.13",
```

Comment:
related changes to react upgrade. latest storybook version 6.5+ accepts react 18 in their peer dependency. For some reason, the set of tools (like addons) are not 100% compatible with react v18. For example, some core addons still uses deps that does not support react v18. 

According to storybook, their official support for react v18 will be available in the next major version v7. Took a look into v7 pre-release, ran into issues with typescript...

Despite the peer dep compatibility for react v18 in storybook v6.5, i was able to do the upgrade and run storybook. It's just that storybook renders the stories with react v17. There is warning about it, also some additional warnings during `npm i` about the conflict when resolving peer dep for react (from core addons).

Example warning in the console when running storybook:
```
ReactDOM.render is no longer supported in React 18. Use createRoot instead. Until you switch to the new API, your app will behave as if it’s running React 17. Learn more: https://reactjs.org/link/switch-to-createroot
```

What are your thoughts about this? We could wait till storybook v7 is stable to upgrade this.

Reply:
@ljiang-ti Since the impact is fairly isolated -- and it seems like it technically still works (albeit as v17) -- I think we can leave the implemented upgrade for now, and upgrade storybook to v7 when it releases.

Reply:
that sounds good to me